feat: Add register for demo page (#455)
cds-snc/gcds-docs: src/en/register-for-a-demo.md

---
title: Register for a demo
translationKey: registerfordemo
layout: 'layouts/base.njk'
eleventyNavigation:
  key: registerfordemoEN
  title: Register for a demo
  locale: en
  order: 5
  hideMain: true
templateEngineOverride: njk,md
---

> # Register for a Demo

Reviewer: ```suggestion
# Register for a demo
```

> Looking to learn more about GC Design System before trying it out? Register to one of the GC Design System upcoming demos.

Reviewer: ```suggestion
Looking to learn more about GC Design System before trying it out? Register to an upcoming GC Design System demo.
```

> Looking to learn more about GC Design System before trying it out? Register to one of the GC Design System upcoming demos.

Reviewer: ```suggestion
Looking to learn more about GC Design System before trying it out? Attend an upcoming demo.
```

Can’t find a time that works for you? [Contact us]({{ links.contact }}).

## What to expect

> - Explore GC Design System components, templates and design tokens

Reviewer: ```suggestion
- Explore GC Design System components, templates, and design tokens
```
